test/export/InstanceExporter.test.ts

```diff
@@ -924,6 +924,24 @@ describe('InstanceExporter', () => {
       });
     });
 
+    it('should log a warning when an invalid reference is fixed', () => {
```

Comment:
This situation logs an error, not a warning.

Reply:
I changed the wording of this! I had written the test before I decided which I was going to log. Oops!